incubator/elastic-stack/README.md

```diff
@@ -0,0 +1,44 @@
+# Elastic-stack Helm Chart
+
+This chart installs an elasticsearch cluster with kibana and logstash by default.
+You can optionally disable logstash and install Fluentd if you prefer. It also optionally installs nginx-ldapauth-proxy and elasticsearch-curator.
+
+## Prerequisites Details
+
+* Kubernetes 1.6+
```

Comment:
1.6? Really? I didn't check the dependent chart, but generally, we support the current and previous minor versions of Kubernetes.

Reply:
Ah true.
I copied that from the elasticsearch chart which may have been written when 1.8 was the supported release.
I will update it.